shakenfist/daemons/net.py

```diff
@@ -60,14 +63,17 @@ def _maintain_networks(self):
                 if not n:
                     continue
 
-                n.create()
+                if not n.is_okay():
+                    LOG.info("Daemon:net: maintain_networks() recreating %s",
```

Comment:
This "Daemon:net" bit is a little odd to me. We should be getting the filename from python logging already. I think I'd prefer something matching the style of the other log lines, and we should rethink logging in general in v0.3.